src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -441,6 +441,15 @@ public void VerifyProjectReferencePresent(string projectName, string referencedP
         {
             var projectReferences = VisualStudio.Instance.SolutionExplorer.GetProjectReferences(projectName);
             Assert.Contains(referencedProjectName, projectReferences);
+
+        }
+        protected void InvokeNavigateTo(string text)
```

Comment:
The expectation is that the test class derive from `AbstractEditorTest` thus providing access.